composer.json

```diff
@@ -57,6 +57,11 @@
         ],
         "patchLevel": {
             "drupal/core": "-p2"
+        },
+        "patches": {
+            "drupal/core": {
+                "3096566 - Copy media library styles from Seven to Claro": "https://www.drupal.org/files/issues/2019-12-05/claro-copy_media_library_styles-3096566-19.patch"
```

Comment:
I don't think we should patch this here. Instead, let's patch Drupal core in Lightning Media 4.x.

Reply:
General correctness and cleanliness. The only thing in Lightning that declares a hard dependency on Media Library is Lightning Media, and components should contain the patches that are relevant to themselves. :)

Reply:
Resolved below: https://github.com/acquia/lightning/pull/660#issuecomment-566134760